framework/ui/iconcodes.h

```diff
@@ -187,6 +187,7 @@ class IconCode
         CHORD_SYMBOL = 0xF352,
         GUITAR_BEND = 0xF353,
         GUITAR_TREMOLO_BAR = 0xF355,
+        MULTIMEASURE_REST = 0xF355,
```

Comment:
Somehow I doubt it's good to use the same value as tremolo bar :-).  I'll find out soon enough if there is an actual icon added I guess...

Reply:
0xF355 is the code for multimeasure rest in the icon font. As far as I know, there is no glyph yet for guitar tremolo bar.

Reply:
I'm confused, because I *don't* see what the comment in the header says I should see - a change to MuseScoreIcon.ttf.  And yet I see see a multimeasure rest icon in the Inspector.  Maybe it is for something else.  But, guitar tremolo bar shows that same icon in the Inspector.  So definitely something is not right.

Reply:
Sorry, posts crossed.  So the mmrest icon was already there, with code F355, and this file was just lying when it says that was guitar tremolo?

Reply:
There is already a glyph for mulimeasure rest in the icon font. It is at codepoint 0xF355. And it is not for something else; Tantacrul added it specifically so it could be used as the icon for multimeasure rest in the inspector.

Reply:
OK, thanks.  So I guess someone gets to add the tremolo one and change the number in this file, but not the job of this PR :-)